.github/actions/spell-check/only.txt

```diff
@@ -1,5 +1,6 @@
 /docs/
 ^docs/
 \.md$
+\.rst$
```

Comment:
This is a major change to the configuration. The docs have mostly migrated from `.md` to `.rst`, so without this, documentation coverage hasn't been as good.